src/site.cabal

```diff
@@ -9,17 +9,11 @@ executable site
                   , MissingH
                   , aeson
                   , blaze-html
-                  , clay
```

Comment:
I'm a bit baffled as to how this is working without `clay`, given that it calls `runghc` on `css/Style.cs` which imports it?! But I can confirm that on this branch running `stack runghc css/Style.hs` works on this branch?!

I'm so confused ...

Reply:
I tried to run just `stack build` and that works.

Just tried your command and it works

Reply:
... but how?!

who is providing the `clay` library? It seems it's coming in as a hidden package somehow, but I can't understand how.

Reply:
Ah, I see what's happening

If you run `stack exec ghc-pkg -- list` you'll see that it's using a clay that is coming from your _system_ (i.e. your root `~/.stack` folder).

This has to be a bug ...

Reply:
Long story short, I think we need to leave the `clay` dependency here; otherwise on a clean machine it can't possibly work ...?

Reply:
agreed

Reply:
question remains: is it safe to remove the other dependencies?

Reply:
readded clay

Reply:
definitely; this is the only one that's questionable because it's _not_ referenced, literally, by stack build

it seems like there's a strange bug/feature where `runghc` doesn't care that the module is hidden; if `site.hs` actually tries to _import_ `Clay`, then it fails (as expected).